cmd/runtimetest/main.go

```diff
@@ -576,6 +576,28 @@ func validateMaskedPaths(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateSeccomp(spec *rspec.Spec) error {
+	if spec.Linux == nil || spec.Linux.Seccomp == nil {
+		return nil
+	}
+	t := tap.New()
+	for _, sys := range spec.Linux.Seccomp.Syscalls {
+		if sys.Action == "SCMP_ACT_ERRNO" {
+			for i, name := range sys.Names {
+				if name == "getcwd" {
+					_, err := os.Getwd()
+					if err == nil {
+						t.Diagnostic("Syscall action ERRNO can not be properly applied in the runtime")
```

Comment:
Maybe the diagnostic could be:

> getcwd did not return an error

Once #308 lands with a local test, that test will be the `%s syscall returns errno` you have in the skip below, so the total output will be:

```
not ok … - getcwd syscall returns errno
# getcwd did not return an error
```